### PR TITLE
Modify rule S2317: LaYC format

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM

File: rules/S2317/python/rule.adoc [hunk 1]
```diff
@@ -1,19 +1,21 @@
+This rule raises an issue when the exec statement is used.
+
 == Why is this an issue?
 
 Use of the ``++exec++`` statement could be dangerous, and should be avoided. Moreover, the ``++exec++`` statement was removed in Python 3.0. Instead, the built-in ``++exec()++`` function can be used.
```
Comment (jean-jimbo-sonarsource) [inline rules/S2317/python/rule.adoc:5]: I feel we're missing the educational aspect of why it's dangerous to use the `exec` statement and therefore also misses to educate on the potential impact especially on how it complements or differs from S1523

File: rules/S2317/python/rule.adoc [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
 
+The ``++exec++`` statement considered dangerous to use for several reasons:
```
Comment (jean-jimbo-sonarsource) [inline rules/S2317/python/rule.adoc:7]: This is not an exhaustive list, so maybe we can say instead "Use of the exec statement is strongly discouraged for several reasons such as:  "

**Review by jean-jimbo-sonarsource — APPROVED** [review]
Added one last suggestion, otherwise OK

**Pull request merged**